Fix Elevation API and extend Elevation Preview

okimiko opened the pull request:
I added a Info field to the elevation preview, which queries the elevation api with coordinates.

During the implementation I stumbled over various issues:
- pmtile coordinates weren't working (just tiles)
- pixel calculation was not working
- swapped lat / long in coordinate input
- use of static tileSize is not a good idea
- the limitation by zoom is not needed for coordinate api

acalcutt commented on public/templates/data.tmpl:
```diff
@@ -118,24 +123,34 @@
+
+    map.addControl(
+      new ElevationInfoControl({
+        url: baseUrl + "/data/{{id}}/elevation/{z}/{x}/{y}"
```
Instead of adding baseUrl so you think you can use {{public_url}} like the javascript links in the header

okimiko replied:
Yes :)

acalcutt replied:
It's ok, you probably got the baseUrl thing from the contour test I posted :-) I didn't think of that then..haha

okimiko replied:
Correct, it's still in my PR to your PR ;-)

acalcutt commented:
I tested this and notice two issue. 

1.) I saw a lot of no data returns, especially around Australia when it was off center to the left of the map. possible when passing the antimeridian

2.) I noticed this issue from before where the terrain window seems to have a white line on the left side.
![image](https://github.com/user-attachments/assets/6978d4eb-84ab-42b7-88ac-23d3bdb1c9d9)

 I changed
` #map { position:absolute; top:0; bottom:0; width:100%; }`
to
` #map {position:absolute;top:0;left:0;right:0;bottom:0;}`
in the data.tmpl and it seemed to fix it.

okimiko commented:
> I tested this and notice two issue.
> 
> 1.) I saw a lot of no data returns, especially around Australia when it was off center to the left of the map. possible when passing the antimeridian
> 
> 2.) I noticed this issue from before where the terrain window seems to have a white line on the left side. 
> 
> I changed ` #map { position:absolute; top:0; bottom:0; width:100%; }` to ` #map {position:absolute;top:0;left:0;right:0;bottom:0;}` in the data.tmpl and it seemed to fix it.

1. Correct, the bounds check was hit there. I removed it and the coordinate conversion works (and the elevation api) works as expected.
2. Fixed, too.
